decentraland/unity-renderer
# fix: Portable Experiences UI issues

## What does this PR change?
Fixes #2143 

When turning on and off the toggle didn't change but the PX dissapeared correctly. Now the status of the UI is synchronized with its PEX in execution.

## How to test the changes?



1. Go to: https://play.decentraland.zone/?renderer-branch=fix/portable-experiences-ui-issues
2. Equip a smart wearable.
3. Notice the PEX is loaded in the Experiences viewer.
4. Activate/Deactivate the the toggle in the PEX.
5. Notice the PEX is being enabled/disabled and the UI's behaviour is working correctly.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
